src/SideNav.js

```diff
@@ -31,12 +31,12 @@ class SideNav extends Component {
     );
 
     return (
-      <div>
+      <Fragment>
```

Comment:
Since we are using `Babel 7` from #714 , we can also use the [`Fragment Short Syntax`](https://reactjs.org/docs/fragments.html#short-syntax)

Hence we can also drop the `Fragment` import at L1